ObfuscarTestNet/TestHelper.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.IO;
+using System.CodeDom.Compiler;
+using System.Text;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace ObfuscarTestNet
+{
+    static class TestHelper
+    {
+        public static string InputPath = Path.Combine("..", "..", "..", "Input");
+
+        private static int count;
+
+        public static string OutputPath
+        {
+            get { return Path.Combine("..", "..", "Output", count++.ToString()); }
+        }
+
+        public static void CleanInput()
+        {
+            // clean out inputPath
+            try
+            {
+                //foreach (string file in Directory.GetFiles(InputPath, "*.dll"))
+                //File.Delete(file);
+            }
+            catch
+            {
+            }
```

Comment:
delete method?

Reply:
Deleted. It was copied from the original test project.